book/variables_and_subexpressions.md

```diff
@@ -1,31 +1,133 @@
 # Variables and Subexpressions
 
-There are two types of evaluation expressions in Nushell: variables and subexpressions. You know that you're looking at an evaluation expression because it begins with a dollar sign (`$`). This indicates that when Nushell gets the value in this position, it will need to run an evaluation step to process the expression and then use the resulting value. Both evaluation expression forms support a simple form and a 'path' form for working with more complex data.
+There are two types of evaluation expressions in Nushell: variables and subexpressions.
+You know that you're looking at an evaluation expression because it begins with a dollar
+sign (`$`). This indicates that when Nushell gets the value in this position, it will
+need to run an evaluation step to process the expression and then use the resulting 
+value. Both evaluation expression forms support a simple form and a 'path' form for
+working with more complex data.
 
 ## Variables
 
-The simpler of the two evaluation expressions is the variable. During evaluation, a variable is replaced by its value.
+The simpler of the two evaluation expressions is the variable. During evaluation, a
+variable is replaced by its value. After creating a variable, we can refer to it using
+`$` followed by its name.
```

Comment:
```suggestion
The simpler of the two evaluation expressions is the variable. During evaluation, a variable is replaced by its value. After creating a variable, we can refer to it using `$` followed by its name.
```

Can you please fix this and the other places where a sentence or paragraph was split into multiple lines?

Reply:
on it